sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastHashJoinExec.scala

```diff
@@ -453,6 +494,47 @@ case class BroadcastHashJoinExec(
     val (keyEv, anyNull) = genStreamSideJoinKey(ctx, input)
     val (matched, checkCondition, _) = getJoinCondition(ctx, input)
     val numOutput = metricTerm(ctx, "numOutputRows")
+    val isLongHashedRelation = broadcastRelation.value.isInstanceOf[LongHashedRelation]
+
+    // fast stop if isOriginInputEmpty = true
+    // whether isNullAwareAntiJoin is true or false
```

Comment:
Same comment as above, is this NAAJ highlight needed here ?

Reply:
done

Reply:
remove the hightlight